config.go

```diff
@@ -42,19 +45,33 @@ type Config struct {
 	linkedFiles      []string
 }
 
+func (c *Config) interfaceAddress(iface string, port int) string {
```

Comment:
The name `interfaceAddress` does not look good to me in this case. Also the parameter `iface`.

Since it takes hole address via the `iface` parameter,  it should not be named `iface`. `addr` would be better in this case.

Also `interfaceAddress` returns the whole address and takes an address and port as param. `listenAddress` or `getListenAddress` would be better I think. 

What do you think?

Reply:
Yeah good points about the naming, I've gone for `addrOrPort` for the private helper method.